examples/onnxrt/nlp/huggingface_model/code_detection/quantization/ptq_static/main.py

```diff
@@ -0,0 +1,347 @@
+        "--train_data_path",
+        default=None,
+        type=str,
+        help="An optional input training data file to evaluate the perplexity on (a text file).",
+    )
+    parser.add_argument(
+        "--data_path",
+        default=None,
+        type=str,
+        help="An optional input evaluation data file to evaluate the perplexity on (a text file).",
+    )
+    parser.add_argument(
+        "--model_name_or_path",
+        default=None,
+        type=str,
+        help="The model checkpoint for weights initialization.",
+    )
+    parser.add_argument(
+        "--model_path", default=None, type=str, help="The onnx model path."
+    )
+    parser.add_argument("--benchmark", action="store_true", default=False)
+    parser.add_argument(
+        "--fine_tune",
+        action="store_true",
+        default=False,
+        help="whether fine tune the model",
+    )
+    parser.add_argument(
+        "--tune", action="store_true", default=False, help="whether quantize the model"
+    )
+    parser.add_argument(
+        "--output_model", type=str, default=None, help="output model path"
+    )
+    parser.add_argument(
+        "--mode", type=str, help="benchmark mode of performance or accuracy"
+    )
+    parser.add_argument("--batch_size", default=1, type=int, help="batch size")
+    parser.add_argument(
+        "--quant_format",
+        type=str,
+        default="QOperator",
+        choices=["QOperator", "QDQ"],
+        help="quantization format",
+    )
+    args = parser.parse_args()
+
+    # fine tune
+    if args.fine_tune:
+        fine_tune(args)
+
+
+    def eval_func(model):
+        session = ort.InferenceSession(
+            model.SerializeToString(), providers=ort.get_available_providers()
+        )
+        ort_inputs = {}
+        len_inputs = len(session.get_inputs())
+        inputs_names = [session.get_inputs()[i].name for i in range(len_inputs)]
+        all_labels = []
+        all_preds = []
+        import tqdm
+
+        for idx, (inputs, labels) in tqdm.tqdm(enumerate(dataloader)):
+            if not isinstance(labels, list):
+                labels: List[np.array] = [labels]  # List[shape: bs]
+            inputs = inputs[:len_inputs]
+            for i in range(len_inputs):
+                ort_inputs.update({inputs_names[i]: inputs[i]})
+            predictions: List[np.array] = session.run(
+                None, ort_inputs
+            )  # List[# shape, (bs, 2)]
+            predictions = [np.argmax(p, axis=1) for p in predictions]
+
+            all_labels += labels
+            all_preds += predictions
+            np.mean(
+                np.concatenate(all_labels, 0) == (np.concatenate(all_preds, 0))
+            )  # [:,0]>0.5))
+        label_flatten = np.concatenate(all_labels, 0)
+        preds_flatten = np.concatenate(all_preds, 0)
+        correct_count = np.sum(label_flatten == preds_flatten)
+        acc = correct_count / len(label_flatten)
+        print(acc)
```

Comment:
remove the print line

Reply:
Removed by [0c26397](https://github.com/intel/neural-compressor/pull/1121/commits/0c2639750f33f6b306544dba11175291c4ad691d).